src/client/localplayer.cpp

```diff
@@ -884,30 +865,30 @@ void LocalPlayer::old_move(f32 dtime, Environment *env, f32 pos_max_d,
 	if (control.sneak && m_sneak_node_exists &&
 			!(fly_allowed && player_settings.free_move) && !in_liquid &&
 			physics_override_sneak) {
-		f32 maxd = 0.5 * BS + sneak_max;
+		f32 maxd = 0.5f * BS + sneak_max;
 		v3f lwn_f = intToFloat(m_sneak_node, BS);
 		position.X = rangelim(position.X, lwn_f.X - maxd, lwn_f.X + maxd);
 		position.Z = rangelim(position.Z, lwn_f.Z - maxd, lwn_f.Z + maxd);
 
 		if (!is_climbing) {
 			// Move up if necessary
-			f32 new_y = (lwn_f.Y - 0.5 * BS) + m_sneak_node_bb_ymax;
+			f32 new_y = (lwn_f.Y - 0.5f * BS) + m_sneak_node_bb_ymax;
 			if (position.Y < new_y)
 				position.Y = new_y;
 			/*
 				Collision seems broken, since player is sinking when
 				sneaking over the edges of current sneaking_node.
 				TODO (when fixed): Set Y-speed only to 0 when position.Y < new_y.
 			*/
-			if (m_speed.Y < 0)
-				m_speed.Y = 0;
+			if (m_speed.Y < 0.0f)
+				m_speed.Y = 0.0f;
 		}
 	}
 
-	// this shouldn't be hardcoded but transmitted from server
-	float player_stepheight = touching_ground ? (BS * 0.6) : (BS * 0.2);
+	// TODO: This shouldn't be hardcoded but decided by the server
+	float player_stepheight = touching_ground ? (BS * 0.6f) : (BS * 0.2f);
 
-	v3f accel_f = v3f(0, 0, 0);
+	v3f accel_f = v3f();
```

Comment:
Superfluous. Check for further `= v3f()` occurrences.

Reply:
Removed, thanks